voila/exporter.py

```diff
@@ -114,6 +114,13 @@ async def generate_from_notebook_node(
         )
         self.register_filter("highlight_code", highlight_code)
 
+        # Why do we need this? Should we make this configurable?
```

Comment:
```suggestion
```
Output can be rendered only if its flag `trusted` is `true`, and JupyterLab handles this flag from the frontend.

Reply:
I meant that, maybe, it may be missing in the nbconvert execute logic. I wonder why we need to manually mark outputs as trusted ourselves.